src/index.html.ejs

```diff
@@ -17,7 +17,7 @@
 <body>
   <div class="header">
     <!-- FIXME images alias isn't getting picked up / translated by webpack -->
-    <img class="logo" src="~images/logo.png" />
+    <img class="logo" src="<%= require('images/logo.png') %>" />
```

Comment:
Nope, that's fine. You don't need the ~ if you require the asset, just if you are using the alias in other contexts (like @import, font url's, and image src=""). 

Turns out we need to require here because this template doesn't get run through any webpack loaders that would scan and replace such things normally. Looked at some people who were trying to wire it so that these templates get processed by your normal webpack loader rules, but doesn't seem worth it in our case.